tests/robot-tests/tests/admin_and_public/bau/release_and_publication_role_permissions.robot

```diff
@@ -0,0 +1,358 @@
+*** Settings ***
+Resource    ../../libs/admin-common.robot
+Library     ../../libs/admin_api.py
+
+Force Tags  Admin  Local  Dev  AltersData  Footnotes
+
+Suite Setup       user signs in as bau1
+Suite Teardown    user closes the browser
+
+*** Variables ***
+${PUBLICATION_NAME}  UI tests - publication_owner %{RUN_IDENTIFIER}
+${RELEASE_TYPE}      Academic Year 2025/26
+${RELEASE_NAME}      ${PUBLICATION_NAME} - ${RELEASE_TYPE}
+${SUBJECT_NAME}      UI test subject
+
+*** Test Cases ***
+Create new publication and release via API
+    [Tags]  HappyPath
+    ${PUBLICATION_ID}=  user creates test publication via api   ${PUBLICATION_NAME}
+    user create test release via api  ${PUBLICATION_ID}   AY    2025
+
+Navigate to admin dashboard and assert publication is present
+    [Tags]  HappyPath
+    user goes to url  %{ADMIN_URL}
+    user selects theme and topic from admin dashboard  %{TEST_THEME_NAME}  %{TEST_TOPIC_NAME}
+    user waits until page contains accordion section   ${PUBLICATION_NAME}  120
+
+Navigate to manage users page as bau1
+    [Tags]  HappyPath
+    user goes to url  %{ADMIN_URL}/administration/users
+    user checks table column heading contains  1  1  Name
+    user checks table column heading contains  1  2  Email
+    user checks table column heading contains  1  3  Role
+    user checks table column heading contains  1  4  Actions
+
+Assert that test users are present in table
+    [Tags]  HappyPath
+    user checks results table row heading contains  1  1  Analyst1 User1
+    user checks results table row heading contains  2  1  Analyst2 User2
+    user checks results table row heading contains  3  1  Analyst3 User3
+    user checks results table row heading contains  4  1  Bau1 User1
+    user checks results table row heading contains  5  1  Bau2 User2
+
+
+    user checks results table cell contains  1  1  	ees-analyst1@education.gov.uk
+    user checks results table cell contains  1  2  	Analyst
+    user checks results table cell contains  1  3  	Manage
+
+    user checks results table cell contains  2  1  	ees-analyst2@education.gov.uk
+    user checks results table cell contains  2  2  	Analyst
+    user checks results table cell contains  2  3  	Manage
+
+    user checks results table cell contains  3  1  	ees-analyst3@education.gov.uk
+    user checks results table cell contains  3  2  	Analyst
+    user checks results table cell contains  3  3  	Manage
+
+    user checks results table cell contains  4  1  	ees-bau1@education.gov.uk
+    user checks results table cell contains  4  2  	BAU User
+
+    user checks results table cell contains  4  3  	Manage
+
+    user checks results table cell contains  5  1  	ees-bau2@education.gov.uk
+    user checks results table cell contains  5  2  	BAU User
+    user checks results table cell contains  5  3  	Manage
+
+Give Analyst1 User1 publication owner access
+    [Tags]  HappyPath
+    # user waits until h1 is visible  Users  60
```

Comment:
I'd remove these commented out lines if they're not being used. And the comment on line 71 too.